File: 7.cs
using System;

class Program
{
    static string ToTime(int seconds)
    {
        if (seconds < 0)
        {
            throw new ArgumentException("Input must be a non-negative integer.");
        }

        int hours = seconds / 3600;
        int minutes = (seconds % 3600) / 60;
        int remainingSeconds = seconds % 60;

        string formattedHours = hours.ToString().PadLeft(2, '0');
        string formattedMinutes = minutes.ToString().PadLeft(2, '0');
        string formattedSeconds = remainingSeconds.ToString().PadLeft(2, '0');

        return $"{formattedHours}:{formattedMinutes}:{formattedSeconds}";
    }

    static void Main()
    {
        int seconds = 70;
        Console.WriteLine(ToTime(seconds));
    }
}

File: 7.php
<?php

$seconds = 60;
function to_time(int $seconds): string {

  if (!is_int($seconds)) {
    throw new InvalidArgumentException("Input must be an integer.");
  }

  $hours = floor($seconds / 3600);
  $minutes = floor($seconds / 60) % 60;
  $seconds = $seconds % 60;

  $hours = str_pad($hours, 2, "0", STR_PAD_LEFT);
  $minutes = str_pad($minutes, 2, "0", STR_PAD_LEFT);
  $seconds = str_pad($seconds, 2, "0", STR_PAD_LEFT);

  return "$hours:$minutes:$seconds";
}


echo to_time($seconds); 
?>
